dozer/cogs/toa.py

```diff
@@ -74,17 +74,15 @@ def __init__(self, bot):
     async def team(self, ctx, team_num: int):
         """Get information on an FTC team by number."""
         res = await self.parser.req("team/" + str(team_num))
-
-        if len(json.loads(res)) == 0:
+        team_data = json.loads(res)[0]
+        if len(team_data) == 0:
             await ctx.send("This team does not have any data on it yet, or it does not exist!")
             return
 
-        team_data = json.loads(res)[0]
-
         e = discord.Embed(color=embed_color)
         e.set_author(name='FIRST® Tech Challenge Team {}'.format(team_num),
                      url='https://www.theorangealliance.org/teams/{}'.format(team_num),
-                     icon_url='https://pbs.twimg.com/profile_images/1049159734249623553/SZ34vdcC_400x400.jpg')
+                     icon_url='https://theorangealliance.org/favicon.ico')
```

Comment:
Discord web and Android (at least) don't render `.ico` files properly. Web and shows an empty space where the icon is, and Android shows a dark gray square.

Reply:
I've changed it already to png